### PR TITLE
feat: update to V8 9.7

### PR DESCRIPTION
**Pull request by lucacasonato**
This commit updates the rusty_v8 to 0.34.0. This commit also adds
the required typings for the new Array#findLast and Array#findIndexLast
methods.

### CODE REVIEW

File: cli/dts/lib.esnext.array.d.ts [hunk 1]
```diff
@@ -21,14 +21,60 @@ interface Array<T> {
...
+   * Returns the value of the last element in the array where predicate is true, and undefined
+   * otherwise.
+   * @param predicate find calls predicate once for each element of the array, in ascending
+   * order, until it finds one where predicate returns true. If such an element is found, find
+   * immediately returns that element value. Otherwise, find returns undefined.
+   * @param thisArg If provided, it will be used as the this value for each invocation of
+   * predicate. If it is not provided, undefined is used instead.
+   */
+  findLast<S extends T>(predicate: (this: void, value: T, index: number, obj: T[]) => value is S, thisArg?: any): S | undefined;
+  findLast(predicate: (value: T, index: number, obj: T[]) => unknown, thisArg?: any): T | undefined;
```
Comment (kitsonk) [inline cli/dts/lib.esnext.array.d.ts:35]: you need to repeat the comment block on the overload for hover to work property.

**Review by kitsonk — APPROVED** [review]
LGTM, one minor comment

File: cli/dts/lib.esnext.array.d.ts (hunk 1 above)
Comment (lucacasonato) [reply cli/dts/lib.esnext.array.d.ts:35]: Copied this from lib.es2015.core.d.ts - that has no two comments. Want me to change it anyway?

Comment (kitsonk) [reply cli/dts/lib.esnext.array.d.ts:35]: naw... keeps it consistent then...

**Pull request merged**